_posts/en/2019-07-24-welcome.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Welcome to IT Strategy!"
+author: "The IT Strategy team"
+date: 2019-07-24
+last_modified: 2019-07-24
+ref: welcome
+lang: en
+---
+
+Welcome to our living workspace! Actually, this is the automatically generated website based on the content living in our actual [workspace](https://github.com/sara-sabr/ITStrategy)!
+
+As per the notice that can be found throughout our site, the content you will find all around should all be considered work in progress: there are no guarantees that the ideas, the propositions and even the presentations that we are developing here in the open will make it into production one day.
```

Comment:
Repetition of "all", suggest to remove 2nd one.

Reply:
Also, I believe "proposals" is more accurate than "propositions".

Reply:
fixed!